tools/Dockerfile

```diff
@@ -126,8 +126,16 @@ ADD https://github.com/plexsystems/konstraint/releases/download/v${KONSTRAINT_VE
 RUN mv /tmp/konstraint-linux-amd64 konstraint && chmod +x konstraint
 
 # node
-RUN curl -sL https://deb.nodesource.com/setup_${NODE_VERSION}.x | bash -
-RUN apt-get install -y nodejs && \
+# https://github.com/nodesource/distributions
+RUN set -uex && \
+  apt-get update && \
+  apt-get install -y ca-certificates curl gnupg && \
```

Comment:
apt packages are installed in line 36.
You will find that ca-certificates  and curl are already installed there. Please move gnupg